src/das-employer-favourites.ApplicationServices/Infrastructure/AzureTableStorageAndFatFavouritesRepository.cs

```diff
@@ -11,19 +13,24 @@
 
 namespace DfE.EmployerFavourites.ApplicationServices.Infrastructure
 {
-    public class AzureTableStorageFavouritesRepository : IFavouritesRepository
+    public class AzureTableStorageAndFatFavouritesRepository : IFavouritesReadRepository, IFavouritesWriteRepository
```

Comment:
Could this be better named? e.g. FatFavouritesTableStorageRepository?

Reply:
👍